ios/brave-ios/Sources/Onboarding/WelcomeFocus/FocusSystemSettingsView.swift

```diff
@@ -0,0 +1,148 @@
+        VStack(spacing: 40) {
+          VStack {
+            settingsSystemContentView
+              .background(colorScheme == .dark ? .black : .white)
+          }
+          .clipShape(RoundedRectangle(cornerRadius: 16.0))
+          .frame(maxWidth: 616, maxHeight: 895)
+          .shadow(color: .black.opacity(0.1), radius: 18, x: 0, y: 8)
+          .shadow(color: .black.opacity(0.05), radius: 0, x: 0, y: 1)
+
+          FocusStepsPagingIndicator(totalPages: 4, activeIndex: .constant(3))
+        }
+        .frame(maxWidth: .infinity, maxHeight: .infinity)
+        .background(Color(braveSystemName: .pageBackground))
+      } else {
+        VStack(spacing: 16) {
+          settingsSystemContentView
+          FocusStepsPagingIndicator(totalPages: 4, activeIndex: .constant(3))
+        }
+        .padding(.bottom, 20)
+        .background(Color(braveSystemName: .pageBackground))
+      }
+    }
+    .navigationViewStyle(StackNavigationViewStyle())
+    .navigationBarHidden(true)
+  }
+
+  private var settingsSystemContentView: some View {
+    VStack {
+      VStack {
+        VStack(spacing: 10) {
+          Text(Strings.FocusOnboarding.defaultBrowserScreenTitle)
+            .font(
+              Font.custom("FlechaM-Medium", size: 32)
+            )
+            .dynamicTypeSize(dynamicTypeRange)
+            .fixedSize(horizontal: false, vertical: true)
+            .multilineTextAlignment(.center)
+
+          Text(Strings.FocusOnboarding.defaultBrowserScreenDescription)
+            .font(
+              Font.custom("Poppins-Medium", size: 17)
+            )
+            .dynamicTypeSize(dynamicTypeRange)
+            .fixedSize(horizontal: false, vertical: true)
+            .multilineTextAlignment(.center)
+            .foregroundColor(Color(braveSystemName: .textTertiary))
+        }
+        .padding(.bottom, 24)
+
+        Spacer()
+
+        LottieAnimationView(
+          name: colorScheme == .dark ? "browser-default-dark" : "browser-default-light",
+          bundle: .module
+        )
+        .loopMode(.loop)
+        .resizable()
+        .aspectRatio(contentMode: .fit)
+        .background(Color(braveSystemName: .pageBackground))
+        .clipShape(RoundedRectangle(cornerRadius: 12.0))
+        .overlay(
+          RoundedRectangle(cornerRadius: 12.0)
+            .stroke(Color(braveSystemName: .textTertiary), lineWidth: 1)
+        )
+        .padding(.bottom, 24)
+
+        Spacer()
+
+        VStack(spacing: 24) {
+          Button(
+            action: {
+              if let settingsUrl = URL(string: UIApplication.openSettingsURLString) {
+                UIApplication.shared.open(settingsUrl)
+              }
+
+              Preferences.FocusOnboarding.urlBarIndicatorShowBeShown.value = true
+              shouldDismiss = true
+            },
+            label: {
+              (Text("\(Strings.FocusOnboarding.systemSettingsButtonTitle) ")
+                + Text(Image(systemName: "arrow.right")))
```

Comment:
Images can be placed inside Text as interpolated values, e.g.:

`Text("\(Strings.FocusOnboarding.systemSettingsButtonTitle) \(Image(systemName: "arrow.right")")`

Reply:
Changed https://github.com/brave/brave-core/commit/91c6c912a8207d7f813f771f7e0e16ff5050eb1d